### PR TITLE
fixed background image error and updated result.html page

### PR DESCRIPTION
**Pull request by saurabh28march**
- some changes in result.html

- background image error fixed

- linked external css to index.html

### CODE REVIEW

**Comment by saurabh28march** [comment]
@maze-runnar  please review !

File: app.py [hunk 1]
```diff
@@ -20,8 +20,8 @@
 
 ## data preprocessing calculations
 
-df_train = pd.read_csv("/home/saurabh/Downloads/DrugData/drugsComTrain_raw.csv")  ###please please add path for your data files
-df_test = pd.read_csv("/home/saurabh/Downloads/DrugData/drugsComTest_raw.csv")
+df_train = pd.read_csv("C:\\Users\\Lenovo\\Desktop\\HackthonFB\\drugsComTrain_raw.csv")  ###please please add path for your data files
+df_test = pd.read_csv("C:\\Users\\Lenovo\\Desktop\\HackthonFB\\drugsComTest_raw.csv")
```
Comment (maze-runnar) [inline app.py:24]: unneccesary

**Comment by maze-runnar** [comment]
Add a screenshot of Index page then @SARTHAK1SINGH merge.

**Comment by saurabh28march** [comment]
> Add a screenshot of Index page then @SARTHAK1SINGH merge.

Screenshot (66)

**Comment by SARTHAK1SINGH** [comment]
Awesome bro 💯

**Pull request merged**